atef/tests/test_ophyd.py

```diff
@@ -0,0 +1,63 @@
+import happi
+import pytest
+from pytestqt.qtbot import QtBot
+from qtpy import QtWidgets
+
+from atef.widgets.config.data_active import ActionRowWidget
+from atef.widgets.ophyd import (OphydAttributeData, OphydAttributeDataSummary,
+                                PolledDeviceModel)
+
+
+def test_ophyd_attribute_data(happi_client: happi.Client):
+    """ Pass if OphydAttributeData/Summary creation is successful """
+    dev = happi_client.search()[0].get()
+    OphydAttributeData.from_device_attribute(dev, 'setpoint')
+    odata = OphydAttributeData.from_device_attribute(dev, 'acceleration')
+
+    OphydAttributeDataSummary.from_attr_data(odata)
+
+
+# TODO: Figure out how to get sim Enum.  SynGauss has one but requires a device
+# as input ... how to do that in happi?
```

Comment:
I think we could just make a simple enum signal for the test suite here and reference it in happi